assets/component-menu-drawer.css

```diff
@@ -223,23 +223,23 @@ details[open].menu-opening > .menu-drawer__submenu {
 
 @media screen and (max-width: 989px) {
   .header--has-localizations:where(:not(.header--has-social)) .menu-drawer__utility-links {
-    padding: 2rem;
+    padding: 3rem;
   }
 }
 
 .menu-drawer__account {
   display: inline-flex;
   align-items: center;
   text-decoration: none;
-  padding: 1.2rem;
+  padding: 1rem 0rem;
   font-size: 1.4rem;
   color: rgb(var(--color-foreground));
-  margin-bottom: 2rem;
+  margin-bottom: 0rem;
```

Comment:
Nitpicks `padding: 1rem 0;` and `margin-bottom: 0s`

Reply:
I just meant to remove the "rem" itself from the `0rem` values as its redundant and we typically just use `0` in these cases.